docs/content/en/docs/install/k8s.md

```diff
@@ -49,3 +52,45 @@ The basic steps are:
    ```shell
    kubectl version --short
    ```
+
+## Prepare your cluster for KLT
+
+The Keptn Lifecycle Toolkit installs into an existing Kubernetes cluster.
+When setting up a local Kubernetes cluster
+to study or demonstrate the Lifecycle Toolkit,
+you need to provide these components.
+
+Your cluster should include the following:
+
+* A supported version of Kubernetes.
+  See [Supported Kubernetes versions](reqs.md/#supported-kubernetes-versions)
+  for details.
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+* Data source such as
```

Comment:
I would maybe suggest to use the word metric provider. I think it may be more descriptive
```suggestion
* Metric provider such as
```